requirements-dev.txt

```
@@ -27,3 +27,4 @@ pytest~=7.3.1
 python-dateutil~=2.8.2
 packaging~=23.1
 setuptools==78.1.1
+holidays>=0.73
```

Let's bump to:

```suggestion
holidays>=0.83
```

that has all the required regions we need.

---

+QQ: passing tests?